README.md

```diff
@@ -143,6 +143,7 @@ packages on [Maven Central][maven-search].
 | Groups Migration API | [v1](clients/google-api-services-groupsmigration/v1) |
 | Groups Settings API | [v1](clients/google-api-services-groupssettings/v1) |
 | Hangouts Chat API | [v1](clients/google-api-services-chat/v1) |
+| HomeGraph API | [v1](clients/google-api-services-homegraph/v1) |
```

Comment:
This seems to be 404 for now, is there a separate step needed to generate the corresponding bindings?